metrics/README.md

```diff
@@ -0,0 +1,53 @@
+# Metrics Plugins
+
+Fluentd has nine \(9\) types of plugins:
+
+* [Input](../input/)
+* [Parser](../parser/)
+* [Filter](../filter/)
+* [Output](../output/)
+* [Formatter](../formatter/)
+* [Storage](../storage)
+* [Service Discovery](../service_discovery/)
+* [Buffer](../buffer/)
+* [Metrics](./)
+
+This article gives an overview of Metrics Plugin.
```

Comment:
It may be off-topic, should we list all types here? (I felt a redundant) :thinking:

Reply:
Currently every plugin, top of them, we list it.

Reply:
This should be working as pointer or providing overview of Fluentd's plugin type.
I would like to leave as-is for now.
I think that removing this plugin type list is unfriendly for newbies.